src/Admin/AdminInterface.php

```diff
@@ -335,7 +335,7 @@ public function setUniqid($uniqId);
     /**
      * Returns the uniqid.
      *
-     * @return int
+     * @return string
      */
     public function getUniqid();
```

Comment:
should be `getUniqId` and `setUniqId` IMO

Reply:
Maybe the goal was consistency with https://secure.php.net/manual/en/function.uniqid.php ?

Reply:
yeah maybe, no need to care about to long ;-)
Lets keep it